webservices/legal_docs/load_legal_docs.py

```diff
@@ -145,10 +145,17 @@ def index_advisory_opinions():
 
         es = utils.get_elasticsearch_connection()
 
-        result = db.engine.execute("""select DOCUMENT_ID, OCRTEXT, DESCRIPTION,
+        result = db.engine.execute("""SELECT DOCUMENT_ID, OCRTEXT, DESCRIPTION,
```

Comment:
Sorry to be a stick-in-the-mud, but I think it helps immensely if the SQL is formatted with uppercase keywords and lowercase everything else.